lib/events/filelog.go

```diff
@@ -60,9 +62,20 @@ func (cfg *FileLogConfig) CheckAndSetDefaults() error {
 	if !utils.IsDir(cfg.Dir) {
 		return trace.BadParameter("path %q does not exist or is not a directory", cfg.Dir)
 	}
+	if cfg.SymlinkDir == "" {
+		cfg.SymlinkDir = cfg.Dir
+	}
+	if !utils.IsDir(cfg.SymlinkDir) {
+		return trace.BadParameter("path %q does not exist or is not a directory", cfg.SymlinkDir)
+	}
 	if cfg.RotationPeriod == 0 {
 		cfg.RotationPeriod = defaults.LogRotationPeriod
 	}
+	// make sure that rotation period is a multiple of one day
+	rotationDays := cfg.RotationPeriod / (24 * time.Hour)
+	if cfg.RotationPeriod != (rotationDays * 24 * time.Hour) {
+		return trace.BadParameter("rotation period should be expressed in multiples of a single day, got %v instead", cfg.RotationPeriod)
```

Comment:
If this message is gonna be shown to a user, I would add an example (e.g. `24h`, `48h`), otherwise it might not be clear what "multiples of a single day" means.

Also, is that okay that this new restriction might cause teleport not to start after upgrade for users who have a "wrong" value in their config?

Reply:
this value is not exposed in the config right now